code/modules/vending/dinnerware.dm

```diff
@@ -18,8 +18,10 @@
 					/obj/item/reagent_containers/glass/mixbowl = 3, // Yogs -- chef's mixing bowl 
 					/obj/item/reagent_containers/food/condiment/cinnamon = 5, // Yogs -- cinnamon shakers!
 					/obj/item/plate = 10) 
-	contraband = list(/obj/item/kitchen/knife/butcher = 2, // Yogs -- Pan
-					  /obj/item/melee/fryingpan = 2) // Yogs -- Pan
+	contraband = list(/obj/item/kitchen/knife/butcher = 2,
+					  /obj/item/melee/fryingpan = 2,	// Yogs -- Pan
+					  /obj/item/twohanded/bigspoon = 2, // Yogs -- Big spoon
```

Comment:
```suggestion
					  /obj/item/twohanded/bigspoon = 1, // Yogs -- Big spoon
```
Yeah theres two chefs, but is two a good idea?

Reply:
there's two of all the others